code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -101,16 +101,16 @@
 
 /obj/machinery/portable_atmospherics/canister/examine(user)
 	. = ..()
+	if(atom_integrity < max_integrity)
+		. += span_notice("The cracks could be repaired with a [EXAMINE_HINT("welding tool")].")
 	. += span_notice("A sticker on its side says <b>MAX SAFE PRESSURE: [siunit_pressure(initial(pressure_limit), 0)]; MAX SAFE TEMPERATURE: [siunit(temp_limit, "K", 0)]</b>.")
-	. += span_notice("The hull is <b>welded</b> together and can be cut apart.")
+	. += span_notice("The hull could be cut apart with a [EXAMINE_HINT("welding tool")].")
```

Comment:
why change this line's meaning?

Reply:
> why change this line's meaning?

The less wordy and more direct a tooltip is the better. You don't need an 'and' in there, it just makes it harder to read in action.

For a tooltip, you only need the main idea.
In this case, the main idea is 'Can be dismantled'
We know the hull is welded together because the hull is made of metal, we don't need that extra information.
We specify welding tool because that is more direct than "The hull is **welded** together and can be cut apart."

We should cut to the chase. It isn't not lore, but rather a tooltip.

Willing to revert change or even try to find a middle ground, but I personally think this is an overall improvement over the previous tooltip.

Reply:
NOTHING else functions this way. 
![image](https://github.com/tgstation/tgstation/assets/6209658/237d8720-8fd3-4869-9a37-108c976b312f)
![image](https://github.com/tgstation/tgstation/assets/6209658/fe14b402-47a8-4d49-a247-b500f309f0f3)
It's cool that you want to cut to the chase, but it's not consistent with every other thing that mentions welding (and while my memory may be a bit fuzzy, but I think it was actually me that standardized this specifically so it's an implication for flavor reasons like 6 years ago lol.)

Reply:
Fair enough. Middle ground it is. I will revert this change and make a new fork and go through each tooltip. However, let's talk about this a bit first.

How would you prefer I go about rewriting them in a more direct way being the person who standardized this?
I'd like to work with you here a bit so we can come to an agreement without stepping on one another's toes.